android/app-example/src/test/java/com/badoo/ribs/example/rib/foo_bar/FooBarInteractorTest.kt

```diff
@@ -60,10 +61,11 @@ class FooBarInteractorTest {
         val result = PermissionRequester.CheckPermissionsResult(emptyList(), emptyList(), emptyList())
         whenever(permissionRequester.checkPermissions(any(), any())).doReturn(result)
 
-        nodeHelper.startAndExecute {}
-        foobarViewEventEmitter.accept(CheckPermissionsButtonClicked)
+        nodeHelper.moveToStateAndCheck(Lifecycle.State.CREATED) {
+            foobarViewEventEmitter.accept(CheckPermissionsButtonClicked)
```

Comment:
Can we test something inside one of the interactors that's already there instead of adding new field only used for test?